pkgs/os-specific/linux/tuxedo-rs/default.nix

```
@@ -0,0 +1,48 @@
+{ lib
+, fetchFromGitHub
+, rustPlatform
+}:
+let
+
+  src = fetchFromGitHub {
+    owner = "AaronErhardt";
+    repo = "tuxedo-rs";
+    rev = "f356f05d8fe40de3c5121c11a5a9b37047501440";
+    hash = "sha256-RVauCQn8T6q+c7z0nYGtxdV/89IWAGSsqHL2xm0OTgI=";
+  };
+
+in
+rustPlatform.buildRustPackage {
+
+  pname = "tuxedo-rs";
+
+  inherit src;
+
+  # NOTE: OfBorg doesn't like importTOML from a subdirectory
+  version = "0.2.1";
```

`src` can probably be inlined here

```suggestion
  pname = "tuxedo-rs";
  version = "0.2.1";

  inherit src;
```